### ISSUE TITLE
Data Source Label Color Contrast

### ISSUE THREAD

**Issue by nidhi-k3** [aewindle110/DroneWQ#38]
Data source file path labels have insufficient color contrast (3.47:1). Gray text #7f8c8d on white background is too light. Change text color from #7f8c8d to #495057 (much darker gray). 

Locations:

"Algae_Survey" path
"Coastal_Study" path
"Lake_Michigan" path
"Reef_Data" path
"Lake_Erie" path
